doc-content/enterprise-only/openshift/offline-repo-proc.adoc

```diff
@@ -27,19 +50,22 @@ ifeval::["{context}"!="openshift-immutable"]
 Alternatively, if you use a custom external repository (for example, Nexus) for your services, you can use the same repository as a mirror repository.
 endif::[]
 +
+endif::offline_onprem[]
 . On the computer that has an outgoing connection to the public Internet, complete the following steps:
 .. Download the latest version of the http://release-engineering.github.io/offliner/[Offliner tool].
 .. Download the `{PRODUCT_FILE}-offliner.txt` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
 .. Enter the following command to download the required artifacts using the Offliner tool:
 +
 [subs="attributes,verbatim,macros"]
 ----
-java -jar offliner-<version>.jar -r https://maven.repository.redhat.com/ga/ -d /home/user/temp rhpam-7.3.1-offliner.txt
+java -jar offliner-<version>.jar -r https://maven.repository.redhat.com/ga/ -d /home/user/temp {PRODUCT_FILE}-offliner.txt
 ----
 +
 Replace `/home/user/temp` with an empty temporary directory on the computer and `<version>` with the version of the Offliner tool.
```

Comment:
In this example, replace...
Is 'on the computer' necessary? We are only working with one computer at the stage, right?

Reply:
I suppose that `offliner-<version>.jar` will be present in the next release download section. correct?
What surprised me is that you are suggesting to create the whole offline repo from scratch, instead of leveraging the existing one, e.g. `Red Hat Process Automation Manager 7.3.1 Maven Repository`.

Reply:
Downloading the offliner tool is the previous step. And that repository does not contain all dependencies, nly the ones we make that's actually a minority.